docs/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
```

Comment:
I'm still somewhat skeptical about the longevity of this repo so it's not very relevant, but _if_ we wanted to keep this code around in the long run, wikimedia code style should be applied eventually.